recipes/icu/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "56.1" %}
+{% set version_under = version.replace(".", "_") %}
+
+package:
+  name: icu
+  version: {{ version }}
+
+source:
+  fn:  icu4c-{{ version_under }}-src.tgz
+  url: http://download.icu-project.org/files/icu4c/{{ version }}/icu4c-{{ version_under }}-src.tgz
+  md5: c4a2d71ff56aec5ebfab2a3f059be99d
+  patches:
+    # include an extra header when using mingw
+    - icu4c-4_9_1-mingw-w64-mkdir-compatibility.patch  # [win]
+    # omit lib prefix on libraries
+    - icu-config.patch                                 # [win]
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python
+
+test:
+  files:
+    - gb-18030-2000.ucm  # from http://source.icu-project.org/repos/icu/data/trunk/charset/data/ucm/gb-18030-2000.ucm
+    - de.txt          # from ICU's data folder
+  commands:
+    # Libraries
+    {% set libs = [
+       "libicudata",
+       "libicui18n",
+       "libicuio",
+       "libicutest",
+       "libicutu",
+       "libicuuc"
+     ] %}
+    {% for each_lib in libs %}
+    - test -f $PREFIX/lib/{{ each_lib }}.a                                                 # [unix]
+    - test -f $PREFIX/lib/{{ each_lib }}.{{ version }}.dylib                               # [osx]
+    - test -f $PREFIX/lib/{{ each_lib }}.so.{{ version }}                                  # [linux]
+    {% endfor %}
+
+    # CLI tests
+    - genbrk --help              # [unix]
+    - gencfu --help              # [unix]
+    - gencnval --help            # [unix]
+    - gendict --help             # [unix]
+    - icuinfo --help             # [unix]
+    - icu-config --help          # [unix]
+    - makeconv gb-18030-2000.ucm # [unix]
```

Comment:
I take this back. I didn't see the `run_test.sh`. Please disregard. Sorry for the noise.